docs/move_model.md

```diff
@@ -0,0 +1,17 @@
+# Move a model to other workspace
+
+The InnerEye models on AzureML are composed of two parts in order to reproduce the same settings at inference and
+training time:
+
+- Model: The model is registed in the AzureML registry and contains the code used at training time and pytorch
+  checkpoint
+- Environment: The Azure ML environment used to train the model. This contains the docker image with all the
+  dependencies that were used for training
+
+If you want to export a model from one Workspace to another you can use:
+
+- Export to
```

Comment:
I think it would be good to say more here what that does. 
- Firstly, which workspace is the model taken from? Is it the one specified on the commandline? You could add comments about how to find the required argument values from settings.yml.
- Where is the model written to, and in which form?